### DISCUSSION TITLE
memory issue

### DISCUSSION THREAD

**Discussion by graz68a** [lllyasviel/Fooocus#3134] [Q&A] [answered]
Hello

I upgraded my server from 16GB to 48 GB (from 8+8 to 8+8 and 16+16) . Now with 48GB loading a model is much more slower than with 16GB . Is it normal ? I upgraded RAM hoping in a speed improvements , on the countrary now Foocus is much more slower . Am I doing something wrong ?

Thank you

**Comment by mashb1t** [comment]
The performance can in most cases be increased by upgrading the GPU. Does your upgraded RAM have more CAS latency?

**Comment by Rogala** [comment, marked as answer]
Try to read the motherboard manual, basically if it has 4 slots for RAM, the first two work faster than the second, install based on the manual to the board to start with 16x2 in these slots and test Fooocus, then add to the second 8x2. And realize that RAMs with different frequencies and timings work on the lowest characteristic of all 4 cards.
